src/main/java/uk/ac/cam/cl/dtg/isaac/dos/GameboardDO.java

```diff
@@ -277,10 +258,27 @@ public final void setTags(final Set<String> tags) {
         this.tags = tags;
     }
 
+    public List<GameboardContentDescriptor> getContents() {
+        return contents;
+    }
+
+    public void setContents(List<GameboardContentDescriptor> contents) {
+        this.contents = contents;
+    }
+
     @Override
     public String toString() {
-        return "GameboardDTO [id=" + id + ", title=" + title + ", questions=" + questions + ", wildCard=" + wildCard
-                + ", wildCardPosition=" + wildCardPosition + ", creationDate=" + creationDate + ", gameFilter="
-                + gameFilter + ", ownerUserId=" + ownerUserId + ", creationMethod=" + creationMethod + ", tags=" + tags + "]";
+        return "GameboardDO [" +
+                "id='" + id + '\'' +
+                ", title='" + title + '\'' +
+                ", contents=" + contents +
+                ", wildCard=" + wildCard +
+                ", wildCardPosition=" + wildCardPosition +
+                ", creationDate=" + creationDate +
+                ", gameFilter=" + gameFilter +
+                ", ownerUserId=" + ownerUserId +
+                ", creationMethod=" + creationMethod +
+                ", tags=" + tags +
+                ']';
```

Comment:
These were auto generated by IntelliJ, do we need to keep the same format? We don't do that much analysis of logs which is where consistency would be important. The main difference is the quoting of strings.

Reply:
No, they are used for caching headers mainly, but so long as they are consistent for a given gameboard for a given API version it will be fine (we don't do cross-version API requests, so only cookies have to be consistent across versions). They rarely get used in logs. One recurring problem is that this style risks generating things like `{HashMap@10952}` for non-string things, which means the caching headers _always_ fail even though the contents of the map/object is identical (because the memory address won't be the same twice, even if the board is the same). I don't think we should be _too_ worried here right now